app/javascript/elm/src/Tooltip.elm

```diff
@@ -68,4 +68,4 @@ typeToggleFilter =
 
 
 activeToggleFilter =
-    TooltipText "Fixed sessions that have submitted a measurement in the past hour are considered active. Fixed sessions that have not submitted measurements in the past hour are considered dormant."
+    TooltipText "Fixed sessions that have submitted a measurement in the past 4 hours are considered active. Fixed sessions that have not submitted measurements in the past 4 hours are considered dormant."
```

Comment:
is this change because of the OpenAQ sensor or in general, independently from it?

Reply:
It's because before `ACTIVE_FOR` was 1 hour. We did change it to 4 hours because OpenAQ lags behind with adding measurements to their platform. That is they need a few hours to add measurements. Thus, we extended the timeframe in which we consider fixed sessions to be active to include most of them.